ros/src/tl_detector/tl_detector.py

```diff
@@ -165,14 +178,12 @@ def get_light_state(self, light):
     def process_traffic_lights(self):
```

Comment:
I had to look at your code here to see what you were doing; at first I was thinking you were working on my part, but then I saw what you were going for! I can't wait to see what you do to "zoom in" on the image for me to process - it sounds cool! I am also curious how you plan to handle the case of a "broken" traffic light (when I was playing with the simulator, I saw one light post that had the two left lane lights "out", with on the right-most lane light showing)...